src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseApprovalServiceTests.cs

```diff
@@ -1628,6 +1689,269 @@ await using (var context = InMemoryApplicationDbContext(contextId))
+                
+                userReleaseRoleService.Setup(mock =>
+                        mock.ListUserReleaseRolesByPublication(ReleaseRole.Approver, release.Publication.Id))
+                    .ReturnsAsync(new List<UserReleaseRole>());
+
+
+                var releaseService = BuildService(
+                    contentDbContext: context,
+                    publishingService: publishingService.Object,
+                    contentService: contentService.Object,
+                    preReleaseUserService: preReleaseUserService.Object,
+                    userReleaseRoleService: userReleaseRoleService.Object
+                    );
+                
+                var result = await releaseService
+                    .CreateReleaseStatus(
+                        release.Id, new ReleaseStatusCreateViewModel
+                        {
+                            PublishMethod = PublishMethod.Scheduled,
+                            PublishScheduled = "2051-06-30",
+                            ApprovalStatus = ReleaseApprovalStatus.HigherLevelReview,
+                            LatestInternalReleaseNote = "Test internal note",
+                            NextReleaseDate = new PartialDate { Month = "12", Year = "2077" }
+                        });
+
+                VerifyAllMocks(contentService,
+                    userReleaseRoleService);
+                result.AssertRight();
+
+            }
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                var saved = await context.Releases.Include(r => r.ReleaseStatuses)
+                    .FirstAsync(r => r.Id == release.Id);
+
+                Assert.False(saved.ReleaseStatuses.First().NotifyReleaseApprovers);
+            }
+        }
+
+        [Fact]
+        public async Task CreateReleaseStatus_HigherReview_SendsEmailIfReleaseApprovers()
+        {
+            var release = new Release
+            {
+                Type = ReleaseType.AdHocStatistics,
+                Publication = new Publication { Title = "Test publication" },
+                ReleaseName = "2030",
+                Slug = "2030",
+                Version = 0,
+            };
+
+            var userReleaseRole1 = new UserReleaseRole
+            {
+                User = new User { Id = Guid.NewGuid(), Email = "test@test.com"},
+                Release = release,
+                Role = ReleaseRole.Approver,
+            };
+            
+            var userReleaseRole2 = new UserReleaseRole
+            {
+                User = new User { Id = Guid.NewGuid(), Email = "test2@test.com"},
+                Release = release,
+                Role = ReleaseRole.Approver,
+            };
+
+            var contextId = Guid.NewGuid().ToString();
+
+            await using (var context = InMemoryApplicationDbContext(contextId))
+            {
+                await context.AddRangeAsync(release, userReleaseRole1, userReleaseRole2);
```

Comment:
You could probably remove this saving of these ReleaseRoles to the databases entirely from the test.  Looks like it's a Mock of the IUserReleaseRoleService that's returning them, and so it won;t need to be looking in any DbContexts for data.

Alternatively you could swap out the Mock IUserReleaseRoleService for a real one and pass it the db context you've populated, in which case yes, you would need to persist the data here first :slightly_smiling_face: